### PR TITLE
9-th task, re-upload

### PR DESCRIPTION
**Pull request by AlexeSemenenko**
Then I'll move it to the final project. I can't do this right now because task 8 isn't approved. (redone pr)

### CODE REVIEW

File: dom_task(temp)/postList.js [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+                }
+                else if (param === 'dateFrom'){
+                    returningPosts = returningPosts.filter(post => post.createdAt >= filterConfig.dateFrom);
+                }
+                else if (param === 'dateTo'){
+                    returningPosts = returningPosts.filter(post => post.createdAt < filterConfig.dateTo);
+                }
+                else if (param === 'author'){
+                    returningPosts = returningPosts.filter(post => post.author === filterConfig.author);
+                }
```
Comment (MaksimViarenich) [inline dom_task(temp)/postList.js]: Minor. ```switch...case``` will look better

File: dom_task(temp)/postList.js [hunk 2]
```diff
@@ -0,0 +1,598 @@
...
+    }
+
+    static comparator(first, second) {
+        if (first.createdAt < second.createdAt){
+            return 1;
+        }
+        else if (first.createdAt > second.createdAt){
+            return -1;
+        }
+        else{
```
Comment (MaksimViarenich) [inline dom_task(temp)/postList.js]: Also ```switch...case```
If you don't want to do it, then last else is not needed.

File: dom_task(temp)/postList.js [hunk 3]
```diff
@@ -0,0 +1,598 @@
...
+    //     hashTags: [
+    //         'firstprogramm', 'helloworld'
+    //     ],
+    //     likes: [
+    //         'Иванов Иван', 'Алексей Семененко'
+    //     ]
+    // }
+]);
+
+class PostElements {
```
Comment (MaksimViarenich) [inline dom_task(temp)/postList.js]: Кто у кого списал?)
У Вас с Паланевичем одинаковые ошибки и структура оформления такая же)

**Review by MaksimViarenich — CHANGES_REQUESTED** [review]

File: dom_task(temp)/postList.js (hunk 3 above)
Comment (AlexeSemenenko) [reply dom_task(temp)/postList.js]: Делали вместе в дискорде просто)

File: dom_task(temp)/postList.js (hunk 1 above)
Comment (AlexeSemenenko) [reply dom_task(temp)/postList.js]: I fixed this, but don't know why it doesn't write outdated

**Pull request closed without merging**